orchagent/fdborch.cpp

```diff
@@ -25,8 +26,14 @@ FdbOrch::FdbOrch(DBConnector *db, string tableName, PortsOrch *port) :
 {
     m_portsOrch->attach(this);
     auto consumer = new NotificationConsumer(db, "FLUSHFDBREQUEST");
-    auto fdbNotification = new Notifier(consumer, this);
+    auto fdbNotification = new Notifier(consumer, this, "FLUSHFDBREQUEST");
```

Comment:
>fdbNotification [](start = 9, length = 15)

Confused about fdbNotification vs fdb_notification. What is the diff?

Reply:
Renamed all the variable names for notification related stuff